test/shared_test/test_ortops.cc

```diff
@@ -303,11 +305,20 @@ TEST(ustring, tensor_operator) {
 
   const auto* api_base = OrtGetApiBase();
   const auto* api = api_base->GetApi(ORT_API_VERSION);
-  api->GetAllocatorWithDefaultOptions(&allocator);
+  auto status = api->GetAllocatorWithDefaultOptions(&allocator);
+  ASSERT_TRUE(status == nullptr || api->GetErrorCode(status) == ORT_OK);
+  if (status != nullptr) {
+    api->ReleaseStatus(status);
+  }
+
   Ort::CustomOpApi custom_api(*api);
 
   std::vector<int64_t> dim{2, 2};
-  api->CreateTensorAsOrtValue(allocator, dim.data(), dim.size(), ONNX_TENSOR_ELEMENT_DATA_TYPE_STRING, &tensor);
+  status = api->CreateTensorAsOrtValue(allocator, dim.data(), dim.size(), ONNX_TENSOR_ELEMENT_DATA_TYPE_STRING, &tensor);
+  ASSERT_TRUE(status == nullptr || api->GetErrorCode(status) == ORT_OK);
+  if (status != nullptr) {
```

Comment:
it looks funny ...

Reply:
?

Reply:
there is a warning if the returned status discard. I copied this code from somewhere, but I just noticed it is not good.

Reply:
It looks good to me.